src/SConscript

```diff
@@ -242,10 +242,12 @@ def blobToCpp(data, symbol, cpp_code, hpp_code=None, namespace=None):
     if hpp_code is not None:
         cpp_code('''\
 #include "blobs/{}.hh"
+
```

Comment:
Still some changes in this file that should be dropped. 

BTW, feel free to rebase this branch. No one else is using is IIRC.